client/src/app/pages/archetypes/archetypes-page.tsx

```diff
@@ -154,25 +162,29 @@ const Archetypes: React.FC = () => {
     });
   };
 
-  const { mutate: deleteReview } = useDeleteReviewMutation(
+  const { mutateAsync: deleteReview } = useDeleteReviewMutation(
     onDeleteReviewSuccess
   );
 
   const discardReview = async (archetype: Archetype) => {
-    try {
-      if (archetype.review?.id) {
-        await deleteReview({
-          id: archetype.review.id,
-          name: archetype.name,
+    if (!archetype.review?.id) {
+      return;
+    }
+    deleteReview({
+      id: archetype.review.id,
+      name: archetype.name,
+    })
+      .then(() => {
```

Comment:
Nit-pick: With the promise callback style, the function doesn't need to be `async`.